src/test/utils/webpage-test.ts

```diff
@@ -1,8 +1,9 @@
 import "mocha";
 import {WebPage} from "../../utils/webpage";
+import {Occurrence} from "../../utils/occurrence";
 let assert = require("chai").assert;
 
-function generateDummyWARC(url : string) : object {
+function generateDummyWARC(url : string) {
     return {
```

Comment:
1. The type is implicitly declared by the returned object. Explicit type on function returns is only required for abstract functions and better readability. 

2. That's just a one liner function anyways, inside a test file. Readability isn't affected by the missing type declaration.